calcite-adapter/src/test/java/software/amazon/documentdb/jdbc/query/DocumentDbQueryMappingServiceBasicTest.java

```diff
@@ -859,14 +859,14 @@ void testComplexQuery() throws SQLException {
                 BsonDocument.parse("{\"$match\": {\"Total\": {\"$gt\": 1}}}"),
                 result.getAggregateOperations().get(5));
         Assertions.assertEquals(
-                BsonDocument.parse(
-                        "{\"$project\": {\"testCollection__id\": \"$testCollection__id\", \"renamed\": \"$field\", \"Total\": \"$Total\", \"_id\": 0}}"),
+                BsonDocument.parse("{\"$sort\": {\"field\": 1}}"),
```

Comment:
@andiem-bq Please review the changes to these tests! Do we have an equivalent test against a server for these queries?

Reply:
This particular one looks good to me and is similar to testCompelxQuery in DocumentDbStatementFilterTest.